runtimes/huggingface/requirements-dev.txt

```diff
@@ -0,0 +1,2 @@
+nest_asyncio==1.5.4
```

Comment:
I think these are only needed for the Alibi tests - should we remove this `requirements-dev.txt`?